concepts/nil/about.md

```diff
@@ -53,12 +53,12 @@ end
 ```
 
 This is a bit different when just using the value in an if statement, since in the last examples would both false and nil be treated as false.
-Here only `nil` is treated as falsy, since if it was false it wouldn't have been nil thereby it would have been truthy.
+Here only `nil` is treated as falsey, since if it was false it wouldn't have been nil thereby it would have been truthy.
```

Comment:
```suggestion
Here only `nil` is treated as falsy, since if it was false it wouldn't have been nil thereby it would have been truthy.
```

I won't bother you by tagging all of these, but as far as I can tell "falsy" is far more common than "falsey"

Reply:
I think we have used falsey even though falsy is the most common because of this docs: https://crystal-lang.org/reference/1.13/syntax_and_semantics/truthy_and_falsey_values.html

Seems like the crystal developers calls it falsey so to keep it consistent with how they have wrote it so do I think falsey is better.

Reply:
:+1: